packages/foundry-sdk-generator/src/generate/betaClient/generatePackage.ts

```diff
@@ -128,6 +132,28 @@ export async function generatePackage(
   } catch (e) {
     consola.error(e);
   }
+
+  // Check if anything generated clashes with names of our own types
+
+  const sanitizedOntology = sanitizeMetadata(ontology);
+  let clashCount = 0;
+  for (
+    const type of [
+      ...Object.keys(sanitizedOntology.actionTypes),
+      ...Object.keys(sanitizedOntology.interfaceTypes),
+      ...Object.keys(sanitizedOntology.objectTypes),
+      ...Object.keys(sanitizedOntology.queryTypes),
+      ...Object.keys(sanitizedOntology.sharedPropertyTypes),
```

Comment:
because its a cli, we don't need to do anything about this but i would probably have done something like an inner function instead of the above for:
```
function count(x: Record<string, any>) {
  for (const type of Object.keys(x) {
    if (osdkTypes.has(type)) clashCount++;
  }
}
const clashCount = count(santizedOntology.actionTypes) + count(sanitizedOntology.interfaceTypes) +...;
```
Due to it avoiding making the extra array. Another option may have been do to a for loop over a for loop leveraging `Object.keys(santitizedOntology)` for the outter.

Reply:
Ah I see, I'll modify anyways to use the inner function, I like that better as well